### ISSUE TITLE
RFC: Don't include underline in default toolbar

### ISSUE THREAD

**Issue by sminnee** [silverstripe/silverstripe-framework#6787]
Underlined text for non-links has been an anti-pattern for more than a decade. The CMS editor is set up to help authors avoid making bad choices (e.g. we don't have font/colour pickers).

In this light, should we remove the underline option from the default toolbar? Sites with a specific need for it can add it back in with config.

**Comment by flamerohr** [comment]
Repeating what I had said:
Normally, it's been accepted as *bold* (html `<strong>`) to bring to attention and _italics_ (html `<em>`) for emphasis.

Although having said that, italics have been in contention due to readability and custom font sheets.

Underline has been understood for active links on webpages for a long time

**Comment by adrexia** [comment]
Yes. I've manually removed the option on every site I'm come across, but it would be good for the default tinyMCE config to not include it.

**Comment by flamerohr** [comment]
Pointer: Update the properties in `TinyMCEConfig` I think it's quite straight forward.

**Comment by maxime-rainville** [comment]
@silverstripeux Do you have views on this?

For the record, I agree with disabling the underline option by default. I think it would be safer to remove it in SS5 however as we're technically removing a feature.

**Comment by sachajudd** [comment]
Yup, happy to disable the underline option by default. 
Related to SS5 TinyMCE improvements: https://github.com/silverstripe/silverstripe-admin/issues/653 
[Designs:](https://github.com/silverstripe/silverstripe-admin/issues/653#issuecomment-427696406)
